### PR TITLE
Tom/signup

### PR DESCRIPTION
**Pull request by TomF-K**
What's new:

• Added email input to signup form
• Added nav bar header to signup.html
• Created basic login page login.html

### CODE REVIEW

**Comment by andy-little** [comment]
Ayyyy! Looking good mate! I've made a couple of small changes that you can [view here](https://github.com/TomF-K/DONE/pull/7/commits/a7f4f8e4c701ee04eb5488fe89420a98336f86c2).
Yes, I've fucked up your styling, sorry, because it was looking good!! 

My challenge to you is, can you get it to look how it did previously using this markup? Add whatever classes you need. The main change is the nav. You should only have one `<nav>`for accessibility reasons. Having two like that will be confusing for people who use screen readers. You _can_ actually have a secondary nav, but it needs to be written correctly, and technically this isn't a secondary nav. 

If you get stuck don't spend ages, I can show you a solution.

**Comment by andy-little** [comment]
Also I removed the `<h1>` from the nav. This isn't really a heading. You should have one h1 per page, it should be used to mark the main title for screen readers, not to make the text big... which is a really common mistake that even professionals make.

**Comment by andy-little** [comment]
Otherwise mate it's looking great! Nice work! Let me know when you wanna have a catch up.

File: src/public/styles.css [hunk 1]
```diff
@@ -9,21 +11,41 @@ body {
...
   font-size: 1rem;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: var(--clr-primary);
 }
 
+/* todo.html styling */
```
Comment (andy-little) [inline src/public/styles.css:22]: nice to see comments like this, makes life easier for other developers

File: src/public/styles.css [hunk 2]
```diff
@@ -9,21 +11,41 @@ body {
...
+  font-weight: lighter;
 }
 
 .flex-center {
   display: flex;
-  flex-flow: column wrap;
+  flex-flow: column nowrap;
   align-items: center;
-  justify-content: space-between;
+  justify-content: ;
```
Comment (andy-little) [inline src/public/styles.css]: small mistake but you should remove this line

File: src/public/styles.css [hunk 3]
```diff
@@ -80,3 +108,44 @@ input:checked + label {
 input {
   display: block;
 }
+
+/* signup.html */
+
+header {
```
Comment (andy-little) [inline src/public/styles.css:113]: if you have multiple pages with multiple headers you might want to style them all differently. I'd definitely look into learning about [BEM](https://css-tricks.com/bem-101/) when you get time. Lot's of people use it, but I think even more people use an approximation of it... just take the philosophy and apply it how you see fit kind of thing.

File: src/public/styles.css (hunk 1 above)
Comment (TomF-K) [reply src/public/styles.css:22]: It's the only way I can see where I've put everything!

**Review by TomF-K — COMMENTED** [review]
Feedback/corrections are much appreciated and all make perfect sense. Have now fixed header styling for signup.html and added submit button.

**Review by andy-little — APPROVED** [review]
whoop 🙌 well done. happy merging :)

**Pull request merged**